### ISSUE TITLE
Rancher Chart Version Mismatch

### ISSUE THREAD

**Issue by zackbradys** [rancherfederal/carbide-charts#103]
**Issue Description:** There is a formatting mismatch of the Helm "Chart Version" for Rancher in `carbide-charts/rancher` when comparing to `rancher-latest/rancher` (and the other Rancher Helm Repos). 

In the Helm Chart for Rancher in `rancher-latest/rancher`, the chart version is formatted as `X.X.X`, but in `carbide-charts/rancher`, the chart version is formatted as `vX.X.X`.

**Issue Troubleshooting:**

```bash
[root@ip-10-0-40-140 rocky]# helm search repo rancher-latest/rancher
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
rancher-latest/rancher  2.7.4           v2.7.4          Install Rancher Server to manage Kubernetes clu...
```

```bash
[root@ip-10-0-40-140 rocky]# helm search repo carbide-charts/rancher
NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
carbide-charts/rancher  v2.7.3          v2.7.3          Install Rancher Server to manage Kubernetes clu...
```

**Issue Resolution:**  Update the Chart Version for `carbide-charts/rancher` to match the upstream chart.

**Comment by zackbradys** [comment]
PR https://github.com/rancherfederal/carbide-charts/pull/104 addresses the issue for Rancher 2.7.4

**Comment by atoy3731** [comment]
Fixed for the other versions of rancher in #108
